core/src/test/java/org/apache/iceberg/util/TestLocationUtil.java

```diff
@@ -46,6 +46,21 @@ public void testStripTrailingSlash() {
     assertThat(LocationUtil.stripTrailingSlash(pathWithOnlySlash))
         .as("Should have no trailing slashes")
         .isEmpty();
+
+    String rootPath = "blobstore://";
```

Comment:
can you please separate these all into their own separate test methods? That was we test only a single thing per test method

Reply:
Done